HW7/HW7Q1.py

```diff
@@ -0,0 +1,23 @@
+"""
+Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. 
+Поскольку разобраться в его кричалках не настолько просто, 
+насколько легко он их придумывает, Вам стоит написать программу. 
+Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) 
+в каждой фразе стихотворения одинаковое. 
+Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. 
+Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с клавиатуры. 
+В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+"""
+input = 'пара-ра-рам рам-пам-папам па-ра-па-да '
+word_list = input.lower().split()
+vowels = ['а', 'у', 'о', 'я', 'ы', 'э', 'и', 'е', 'ё']
```

Comment:
Это можно задать просто строкой.
```suggestion
vowels = 'аяуюоеёэиы'
```